ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx

```diff
@@ -118,7 +119,9 @@ const ContentView = ({ failure }: ContentViewProps) => {
           "features.itWallet.unsupportedDevice.error.secondaryAction"
         ),
         onPress: () =>
-          Linking.openURL("https://io.italia.it/documenti-su-io/faq/#n1_12")
+          openWebUrl("https://io.italia.it/documenti-su-io/faq/#n1_12", () =>
+            IOToast.error(I18n.t("global.jserror.title"))
```

Comment:
Prefer using `useIOToast` within functional components. The `IOToast` service is designed for use outside the component tree.

Reply:
Fixed, thanks!